### ISSUE TITLE
Labels

### ISSUE THREAD

**Issue by sandeep-d-patil** [sandeeprockstar/Maskrcnn-Cityscapes#1]
https://github.com/sandeeprockstar/Maskrcnn-Cityscapes/blob/master/cityscapesdataset.py
Labels produced by the code below for first instance of the dataset is 13 labels.
```
self.cityscapesID_to_ind = {
            l.id: self.name_to_id[l.name] for l in csHelpers.labels if l.hasInstances
        }
        instIds = torch.sort(torch.unique(torch.from_numpy(mask)))[0]
        for instId in instIds:
            if instId < 1000:  # group labels
                continue

            # masks = masks == instId
            label = int(instId // 1000)
            label = self.cityscapesID_to_ind[label]
            labels.append(label)

        labels = torch.as_tensor(labels, dtype=torch.int64)
```

while labels produced by the code below for the same instance of the dataset is 24.

`labels = torch.ones((num_objs,), dtype=torch.int64)`

Hence this mismatch should be solved.
